cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html

```diff
@@ -47,36 +47,28 @@
 
     {% from 'molecules/expandable.html' import expandable with context %}
     {% call() expandable(sec_nav_settings) %}
-        {{ _secondary_navigation(nav_items, active_nav_id) | safe }}
+        {% import 'molecules/nav-link.html' as nav_link %}
+        <ul class="o-secondary-navigation_list
+                   o-secondary-navigation_list__parents">
+            {%- for item in nav_items %}
+                <li data-nav-is-active="{{ request.url.startswith(item.url) }}">
+                    {{ nav_link.render(item.title, item.url, true, item.url == request.url) }}
+                    {%- for child in item.children -%}
+                        <ul class="o-secondary-navigation_list
```

Comment:
The `ul` should move outside the loop, otherwise there's a ton of `ul`s. Should also prob wrap all of it in a `if item.children` condition.